packages/richtext-lexical/src/features/indent/server/index.ts

```diff
@@ -7,6 +7,13 @@ export type IndentFeatureProps = {
    * These can be: "paragraph", "heading", "listitem", "quote" or other indentable nodes if they exist.
    */
   disabledNodes?: string[]
+  /**
+   * If false, pressing Tab in the middle of a block such as a paragraph or heading will insert a tabNode.
```

Comment:
Since false is the default, shouldn't we explain what setting it to true does instead?